pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -664,3 +665,21 @@ func enteredCycle(req *internalListUsersRequest) bool {
 	req.visitedUsersetsMap[key] = struct{}{}
 	return false
 }
+
+func (l *listUsersQuery) buildResultsChannel() chan *openfgav1.User {
+	foundUsersCh := make(chan *openfgav1.User, 1)
```

Comment:
Setting the channel size to one would block the channel from receiving until the single result was processed,  creating an unnecessary performance bottleneck. We should probably set this value much higher in cases where there is no results limit.